engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-traces-modal.html

```diff
@@ -0,0 +1,75 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header">Content Host Tracer Management</h4>
+
+  <div data-block="modal-body">
+    <form id="traceActionForm" name="traceActionForm" class="form" method="post" action="/katello/remote_execution">
+      <input type="hidden" name="remote_action" value="service_restart"/>
+      <input type="hidden" name="name" ng-value="traceActionFormValues.service"/>
+      <input type="hidden" name="host_ids" ng-value="traceActionFormValues.hostIds"/>
+      <input type="hidden" name="customize" ng-value="traceActionFormValues.customize"/>
+      <input type="hidden" name="authenticity_token" ng-value="traceActionFormValues.authenticityToken"/>
+    </form>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search" ng-show=tracesearch></div>
+      <span data-block="list-actions" ng-hide="contentHost.readonly">
+        <div bst-modal="performViaRemoteExecution(true)">
+          <div data-block="modal-header" translate>Restart services on the selected Content Hosts ?</div>
+          <div data-block="modal-body" translate>Are you sure you want to restart the services on the content hosts chosen?</div>
```

Comment:
I'm getting a scroll bar on this.  Do you have it too?

![modal_confirmation](https://user-images.githubusercontent.com/22042343/79262432-5158fa00-7e5f-11ea-92bf-798c27a1a4bf.png)

(if not, it may be related to another PR I had npm installed so don't worry about it)

Reply:
"selected content hosts" seems more clear than "content hosts chosen"

Reply:
@jeremylenz  Fixed the wording, the scrollbar looks to be part of the CSS from what I was reading online and to remove it, we would have to change some of that. It looks like it is appearing in other modals as well:

https://nimb.ws/bSMB57

I am fine with filing an issue to fix that up and linking it into the pr, but since it is appearing in other modals, I don't think it should block the PR.

Reply:
@johnpmitsch Could this be related to having the PF4 CSS on our dev boxes?

Reply:
@jeremylenz its possible, but would have to dig in to confirm. You can try pinning back the @theforeman/vendor npm packages to see.